src/main/java/org/sagebionetworks/web/client/widget/entity/editor/TeamSelectEditor.java

```diff
@@ -2,26 +2,31 @@
 
 import java.util.List;
 import java.util.Map;
+
 import org.sagebionetworks.repo.model.principal.TypeFilter;
 import org.sagebionetworks.web.client.widget.WidgetEditorPresenter;
 import org.sagebionetworks.web.client.widget.entity.dialog.DialogCallback;
 import org.sagebionetworks.web.client.widget.search.SynapseSuggestBox;
 import org.sagebionetworks.web.client.widget.search.UserGroupSuggestionProvider;
 import org.sagebionetworks.web.shared.WidgetConstants;
 import org.sagebionetworks.web.shared.WikiPageKey;
+
 import com.google.gwt.user.client.ui.Widget;
 import com.google.inject.Inject;
 
 public class TeamSelectEditor implements WidgetEditorPresenter {
 	private Map<String, String> descriptor;
+	TeamSelectEditorView view;
```

Comment:
now the UI widget is the view (used to be the SynapseSuggestBox, but now the view contains the SynapseSuggestBox)